git_hooks/create_hooks

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+HOOK_DIR=$(git rev-parse --show-toplevel)/.git/hooks
+ln -s -f ../../git_hooks/pre-push $HOOK_DIR/pre-push
```

Comment:
Newline?

Reply:
dont really agree, how strongly do you feel about this